### PR TITLE
IV-CONFIG-1-1: Set config server configuration

### PR DESCRIPTION
**Pull request by guillaume-leroy**
https://trello.com/c/lIIN3CA4/1-iv-config-1-1-create-configuration-server

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+spring:
+  cloud:
+    config:
+      server:
+        native:
+          search-locations: ${CONFIG_DIR_PATH}
+
+  profiles:
+    active: native
```
Comment (Ydrasil) [inline src/main/resources/application.yml]: Is that normal that there is no newline at the end of line ?

Comment (guillaume-leroy) [reply src/main/resources/application.yml]: Yes, I never add new line at the end of file. Is it a problem?

Comment (Ydrasil) [reply src/main/resources/application.yml]: I was juste asking.

Comment (Ydrasil) [reply src/main/resources/application.yml]: The things is that some editor (or configuration) add a newline automatically, that's why I'm asking for this, for avoid potential conflict later 🙂

Comment (guillaume-leroy) [reply src/main/resources/application.yml]: Yeah, I don't know if it is a good practice but I never check that. Sometimes, there is a new line, sometimes not. Do you think it's important to "standardize" this?

**Pull request merged**